pkgs/applications/science/electronics/dataexplorer/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchurl
+, jdk
+, jre
+, ant
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dataexplorer";
+  version = "3.6.2";
+
+  src = fetchurl {
+    url = "mirror://savannah/dataexplorer/dataexplorer-${version}-src.tar.gz";
+    sha256 = "sha256-2e8qeoJh7z/RIowMtAd8PGcMPck5H8iHqel6bW7EQ0E=";
+  };
+
+  nativeBuildInputs = [ jdk ant makeWrapper ];
+
+  buildPhase = ''
+    ant -f build/build.xml dist
+  '';
+
+  doCheck = false;
+  # Missing dependencies (e.g. junit). Does not work.
```

Comment:
Did you try adding junit to checkInputs?

Reply:
Yes and it would still fail, because the Ant build.xml hardcodes some paths to the junit jar file and it seems it uses a different version than available in Nixpkgs. The [pkg in Arch Linux](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dataexplorer) (for an older version) also doesn't run the unit tests.